main.go

```diff
@@ -115,6 +116,59 @@ func (s *server) Plus(ctx context.Context, in *pb.PlusRequest) (*pb.PlusReply, e
 	return &pb.PlusReply{Result: in.A + in.B}, nil
 }
 
+// SayHelloMultiReply streams HelloReply back to the client.
+func (s *server) SayHelloMultiReply(req *pb.HelloRequest, stream pb.Greeter_SayHelloMultiReplyServer) error {
```

Comment:
Better to encode the stream type in the function name, so we don't need to add comment for them